sway/commands/move.c

```diff
@@ -208,31 +248,78 @@ static struct cmd_results *move_in_direction(struct sway_container *container,
 	return cmd_results_new(CMD_SUCCESS, NULL, NULL);
 }
 
+static const char* expected_position_syntax =
```

Comment:
Nitpick: this can be `char[]` instead of `char *`

Reply:
There's another `char *` near the top of this file. I don't really want to go changing that one too. I think if our preference is `char []` then we should replace them all across all commands in one commit.

Reply:
`char[]` is objectively better (see https://www.akkadia.org/drepper/dsohowto.pdf section 2.4.1 Pointers vs. Arrays), so I'd prefer new code not to use `char *` if possible. It's no big deal anyway.